### DISCUSSION TITLE
Why is “ALLSKY_EXTRA” on the SD card

### DISCUSSION THREAD

**Discussion by Adler6907** [AllskyTeam/allsky-modules#37] [Ideas]
Modules like `allsky_dewheater.py` constantly write to `/home/pi/allsky/config/overlay/extra` which is not in RAM like `/home/pi/allsky/tmp`.
Wouldn't it be better to change `ALLSKY_EXTRA="${ALLSKY_OVERLAY}/extra"` to `ALLSKY_EXTRA="${ALLSKY_TMP}/extra"` in `/home/pi/allsky/variables.sh`?

Then in the example the file `allskydew.json` in `/home/pi/allsky/tmp/extra` would be constantly written.

**Comment by Alex-developer** [comment]
They are there to preserve the files times. The file times are used to calculate expiry for data files.

I'll add an option to move it into the memory file system for people that are worried about it

**Comment by Alex-developer** [comment]
Closing this as it will be handled under the issue https://github.com/AllskyTeam/allsky-modules/issues/40
